chainstate/types/src/block_status.rs

```diff
@@ -13,29 +13,54 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use bitflags::bitflags;
+use derive_more::Display;
 use enum_iterator::Sequence;
 use serialization::{Decode, Encode};
 
+// FIXME: which repr should I specify for the enums?
+// (Also, can changing the repr in the future lead to incompatibilities with the existing DB?).
```

Comment:
Yes. In that case we have to upgrade the db storage version and the database has to be re-indexed.